chart/openfaas/values.yaml

```diff
@@ -63,6 +63,33 @@ operator:
       memory: "120Mi"
       cpu: "50m"
 
+# configuration options for the faas-lambda provider
+faaslambda:
+  image: ewilde/faas-lambda:0.1
+  create: false
+  replicas: 1
+  readTimeout : "60s"
+  writeTimeout : "60s"
+  imagePullPolicy : "Always"    # Image pull policy for deployed functions
+  httpProbe: true               # Setting to true will use HTTP for readiness and liveness probe on Pods (incompatible with Istio < 1.1.5)
+  setNonRootUser: false
+  readinessProbe:
```

Comment:
Are you consuming these probe values inside the YAML or the controller?

Reply:
😊 looks like these are referenced in the yaml, but almost certainly not being used by the provider, will double check and remove anything not used in the provider.